### PR TITLE
Automate production deployment

### PR DESCRIPTION
**Pull request by timber-they**
Add script to automate deployment

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# This script automates the following steps:
+# - Tag a version on the main branch
+# - Update the corresponding docker-compose.yml on the prod server
+# - Restart the corresponding docker container on the prod server
+
+# Exit if a command failed
+set -e
+
+packagesUrl="https://github.com/mint-o-badges/badgr-server/pkgs/container/badgr-server"
+productionUrl="openbadges.education"
```
Comment (YouQam) [inline deploy.sh:11]: I'm not too familiar with bash scripts, but should `productionUrl`here be `api.openbadges.education`?

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+else
+    echo "Tagging commit $checksum with version $version..."
+fi
+git tag -a "$version" $checksum -m "$message"
+
+echo "Pushing tags to origin..."
+git push origin --tags
+
+echo -n "Waiting for build to finish..."
+while ! curl -s "$packagesUrl" | grep "$version" > /dev/null; do
```
Comment (jona159) [inline deploy.sh:43]: Do you think we should add something to handle a case where the curl command fails? Can this ```do while``` result in an infinite loop or am too scared maybe?

Comment (timber-they) [reply deploy.sh:43]: What do you mean by fail? Which cases should we consider, in your opinion? E.g. if there's no internet connection, the commands before fail already. If github is down also (and then we'd have different problems). Another possibility is of course that the UI of github changes, thus indeed causing an infinite loop here. But I'm not sure how we'd check this. The alternative I could think of would be to use the github-cli program, but that needs to be installed first. Maybe I can check if I can find some kind of API feature from github for this though...

Comment (timber-they) [reply deploy.sh:43]: Apparently there is an API endpoint for this, but in order to use it you need to have an access token with the `read:packages` scope. We thus have the following options:
1. Add a personal access token with this scope to this public repository
2. Handle the token as a secret, meaning that if someone wants to use the script, they need to add their own access token
3. Leave it as it currently is, not requiring a token

None of these options seems ideal. What do you think?

Comment (jona159) [reply deploy.sh:43]: I guess this is an unlikely edgecase, that we could still adapt to if it were to happen. I will approve of the script as it is

**Review by jona159 — APPROVED** [review]

**Pull request merged**